clusterdock/topologies/cdh/actions.py

```diff
@@ -185,13 +254,21 @@ def start(args):
                 "direct any feedback to our community forum at "
                 "http://tiny.cloudera.com/hadoop-101-forum.")
 
-def restart_cm_server(primary_node):
+def get_cloudera_nodes(cluster):
```

Comment:
Not sure this needs to be a function; you can get the primary and secondary nodes just by accessing the `nodes` attribute of the `primary_node_group` and `secondary_node_group` instances.